examples/spring-boot-with-certmanager-example/README.md

```diff
@@ -0,0 +1,118 @@
+# Spring Boot with Cert-Manager
+
+The purpose of this Spring Boot example is to demonstrate how we can configure a HTTPS/TLS microservice using the Cert-Manager Dekorate extension.
+
+Apart from the Dekorate Kubernetes starter dependency, the application must declare the Cert-Manager dekorate dependency part of the pom.xml file:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>certmanager-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+## Enable HTTPS transport
+
+To enable the HTTPS/TLS transport in Spring Boot, we need to add the following properties:
+
+```
+server.port=8443
+server.ssl.enabled=true
+```
+
+Next, we need to configure the Java PKCS12 Keystore properties that our Spring Boot application will use to get the Server certificate signed and the private key:
+
+```
+server.ssl.key-store-type=PKCS12
+server.ssl.key-store=/path/to/keystore.p12
+server.ssl.key-store-password=the password
+```
+
+Where is the `keystore.p12` Keystore and what is its password? This is where Cert-Manager comes to play. This Java Keystore will be generated by Cert-Manager on the Kubernetes platform as a secret resource. Then, what we need to do is to mount a volume from this generated secret, so the Spring Boot application can read the generated Keystore by Cert-Manager. In the following sections, we'll see how we can instruct Cert-Manager to generate the Keystore and how to configure the application to mount the secret and use it.
```

Comment:
`will be generated by Cert-Manager ` --> `will be generated by the Cert-Manager `